lib/pbench/server/cache_manager.py

```diff
@@ -340,86 +297,55 @@ def subprocess_run(
                     f"{shlex.split(command)[0]} exited with status {process.returncode}:  {process.stderr.strip()!r}",
                 )
 
-    @staticmethod
-    def do_move(src: Path, dest: Path, ctx: Path):
-        """Moves unpacked Tarball from unpacked to incoming directory
+    def unpack(self):
+        """Unpack a tarball into a temporary directory tree
 
-        Args:
-            src: source directory to move contents from.
-            dest: destinations directory to move contents to.
-            ctx: tarball path/unpack directory path. This is only used at
-                the event of an error as a parameter to the CacheManagerError Exception.
-
-        Raises:
-            TarballUnpackError if an exception/failure occurs while unpacking
-            the tarball.
-        """
-        try:
-            shutil.move(src, dest)
-        except Exception as exc:
-            raise TarballUnpackError(
-                ctx,
-                f"Error moving {str(src)!r} to {str(dest)!r}: {str(exc)}",
-            )
+        Unpack the tarball into a temporary cache directory named with the
+        tarball's resource_id (MD5).
 
-    def unpack(self, incoming: Path, results: Path):
-        """
-        Unpack a tarball into the INCOMING directory tree; this assumes that
-        the INCOMING controller directory already exists, which should be
-        ensured by calling this indirectly through the Controller class unpack
-        method.
+        This tree will be used for indexing, and then discarded. As we build
+        out more of the cache manager, it can also be used to build our initial
+        cache map.
 
-        Args:
-            incoming: Controller's directory in the INCOMING tree
-            results: Controller's directory in the RESULTS tree
+        The indexer works off the controller INCOMING directory, assuming the
```

Comment:
Sadly, `controller` is still relevant because the `ARCHIVE` tree needs to work with the companion 0.69 server; but, yeah, the comment eluded my attention. Probably hiding behind a tree...